cmd/inspector/main.go

```diff
@@ -445,6 +463,62 @@ func CreateCSVStats(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+// ObjectHealth gets information about the health of an object on the network
+func ObjectHealth(cmd *cobra.Command, args []string) (err error) {
+	ctx := context.Background()
+
+	i, err := NewInspector(*Addr, *IdentityPath)
+	if err != nil {
+		return ErrArgs.Wrap(err)
+	}
+
+	req := &pb.ObjectHealthRequest{
```

Comment:
FYI, when requesting the object, setting the segment index = -1 will get all segments corresponding to that object. So I will leave it upto you ... as it is good and no comments. If you are working on fixing other comments (if any that comes up), look into merging ObjectHealth & SegmentHealth into one function, saves few lines of code. This is an observation.

Reply:
We want the ability to look up individual segments and objects separately

Reply:
-1 should only go through the loop once